src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -6,6 +6,8 @@
 from azure.cli.core.util import CLIError
 from azure.cli.core.commands.parameters import get_resources_in_subscription
 
+from azure.mgmt.containerregistry.v2017_03_01.models import SkuTier
```

Comment:
Why not use https://github.com/Azure/azure-cli/blob/a6b8fd8a5306a66a96b52d245b13cc96c667fa13/src/azure-cli-core/azure/cli/core/profiles/__init__.py#L50

@derekbekoe